OsmAnd-java/src/main/java/net/osmand/osm/MapPoiTypes.java

```diff
@@ -547,6 +547,9 @@ public void initFromInputStream(InputStream is) {
                                 }
                             }
                         }
+						case "poi_types" -> {
```

Comment:
Formatting

Reply:
fixed but there is a lot of more bad formatting...